go/ephemeral/lib.go

```diff
@@ -86,15 +91,8 @@ func (e *EKLib) ShouldRun(ctx context.Context) bool {
 	willRun := ok || g.Env.GetFeatureFlags().Admin() || g.Env.GetRunMode() == libkb.DevelRunMode || g.Env.RunningInCI()
 	if !willRun {
 		e.G().Log.CDebugf(ctx, "EKLib skipping run uid: %v", uid)
-		return false
-	}
-
-	oneshot, err := g.IsOneshot(ctx)
-	if err != nil {
-		g.Log.CDebugf(ctx, "EKLib#ShouldRun failed: %s", err)
-		return false
 	}
-	return !oneshot
```

Comment:
Are we sure there are no other `ShouldRun` callers that are relying on this check? Should we leave it in both places?

Reply:
`ShouldRun` is only used to feature flag this but will be removed once we release so we are good to remove it here